docs/eventing/channels/subscriptions.md

```diff
@@ -7,6 +7,13 @@ showlandingtoc: "false"
 
 After you have created a channel and a sink, you can create a subscription to enable event delivery.
 
+The subscription consists of a Subscription object, which specifies the channel and the sink (also
```

Comment:
Might want to mention the reply here? I see it's mentioned below but for completeness, maybe here too?

Reply:
I think let's open a follow up issue for that @vaikas - it seems like it'll require some explanation or its own topic? WDYT?

Reply:
@abrennan89 sounds like a plan. And I know I owe you some pointers there. I should have more time in the next day or so after the release cuts.